RevenueCatUI/Helpers/ImageLoader.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ImageLoader.swift
+//
+//
+//  Created by Andrés Boedo on 11/29/23.
+//
+
+import Combine
+import Foundation
+import UIKit
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+extension URLCache {
+
+    static let imageCache = URLCache(memoryCapacity: 5 * 1000 * 1000, diskCapacity: 30 * 1000 * 1000)
```

Comment:
We'll need to update `PaywallCacheWarming` to use this too, so we'll probably have to expose it from `Purchases`.

Reply:
agreed! I commented the same thing